packages/infrastructure/kube_vault_proxy/main.tf

```diff
@@ -27,6 +27,10 @@ terraform {
   }
 }
 
+locals {
+  path_prefix = replace("${var.path_prefix}/oauth2", "//", "/")
```

Comment:
This _should_ ensure that we don't end up with `//my/path/oauth2`, but maybe this should be a regex-validated input instead.